### PR TITLE
Tips on ssh to crashing app

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:
-
- Help on ssh to a crashing app

Maybe a quick blog post on this too?
-

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/peterb/ssh-tips)

## Security Considerations

This user-facing ssh tip has no security implications.

### CODE REVIEW

File: _docs/management/using-ssh.md [hunk 1]
```diff
@@ -14,9 +14,17 @@ ssh`](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#ssh-comma
...
+
+**Connect to a crashing app**: You may be trying to determine why your application keeps crashing, but it doesn't stay up long enough for an SSH session. This happens because Cloud Foundry detects when an application fails the "health check" (typically by connecting to a TCP port), then recreates the container and runs the start command again. To determine _why_ your start command is failing, you can override the _start command_ and the _health check_ with:
+
+> `cf push -u process -c "sleep 600" ... [your other push options]`
+
+This overrides the port health check with the `process` check, and sets the start comment to just `sleep`. That will give you 10 minutes to `cf ssh` and inspect your container.
```
Comment (mheadd) [inline _docs/management/using-ssh.md]: Do you mean `sets the start command to just sleep` ..?

Comment (mheadd) [reply _docs/management/using-ssh.md]: `command` rather than `comment`

Comment (pburkholder) [reply _docs/management/using-ssh.md]: Indeed. Fixed. Thanks.

**Review by mheadd — APPROVED** [review]
LGTM!

**Pull request merged**